template-sample-app/template/src/components/views/stores/StoresList.js

```diff
@@ -57,9 +57,9 @@ export default function StoresList() {
         id='storesByRevenue'
         title='Stores by revenue'
         dataSource={SOURCE_ID}
-        formatter={numberFormatter}
+        yAxisFormatter={numberFormatter}
         xAxisFormatter={currencyFormatter}
-        operation={AggregationTypes.COUNT}
+        operation={AggregationTypes.SUM}
```

Comment:
It's not corrent. We don't want to SUM, we want to count the number of stores in the range of the histogram

Reply:
Sorry, that was a test and I forgot to remove it

Reply:
Done